### PR TITLE
[HEROKU] feat: add autogenerated heroku spec

### PR DESCRIPTION
**Pull request by cstrnt**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
The current heroku cli spec is lacking a lot of subcommands

**What is the new behavior (if this is a feature change)?**

This is an autogenerated spec created from the source code

**Additional info:**

### CODE REVIEW

**Review by mattschrage — COMMENTED** [review]
Is there anyway to get the short flags as well. Eg.  `-a` for `--app`?

**Comment by cstrnt** [comment]
Sure, I'll try to figure out how it's implemented in the code and I'll try to implement it as well

**Comment by QuiiBz** [comment]
What a huge CLI, congrats on the huge work!

File: dev/heroku.ts [hunk 1]
```diff
@@ -1,122 +1,7079 @@
...
+      name: "domains:remove",
+      description: "remove a domain from an app",
+      args: [
+        {
+          name: "hostname",
+        },
+      ],
+      options: [
+        {
+          name: ["-h", "--help"],
```
Comment (mattschrage) [inline dev/heroku.ts:561]: Seems like the `--help` flags all take arguments? Is this expected?

**Review by mattschrage — APPROVED** [review]
This looks great @cstrnt! Thanks for tackling `heroku` - it's a beast of a CLI and I use it all the time.

File: dev/heroku.ts (hunk 1 above)
Comment (cstrnt) [reply dev/heroku.ts:557]: Nah, you're right. Just fixed this

**Review by QuiiBz — APPROVED** [review]

**Pull request merged**